### PR TITLE
 Dilpreet | Stage | Implement material on voucher page (except PO)

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**

### CODE REVIEW

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html [hunk 1]
```diff
@@ -152,40 +151,38 @@ <h4 class="modal-title bg pull-left">{{ localeData?.page_heading }}</h4>
...
                                             "
-                                            class="text-left"
-                                            name="voucherName{{ entry.uniqueName }}"
-                                            [defaultValue]="entry.voucherNumber"
-                                            [fixedValue]="entry.voucherNumber"
-                                            (click)="clickSelectVoucher(entryIdx, adjustAdvancePaymentForm)"
-                                            required
+                                            *ngIf="voucherApiVersion === 2"
+                                            [cssClass]="'form-control mat-field-border text-left'"
+                                            [name]="'voucherName{{ entry.uniqueName }}'"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html]:
```suggestion
                                            [name]="'voucherName' + entry.uniqueName"
```

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html [hunk 2]
```diff
@@ -202,35 +199,34 @@ <h4 class="modal-title bg pull-left">{{ localeData?.page_heading }}</h4>
...
+                                        <select-field
                                             *ngIf="voucherApiVersion !== 2"
-                                            [disabled]="
+                                            [readonly]="
                                                 shouldDisableEdit(entry) || entry.linkingAdjustment ? 'disabled' : null
                                             "
-                                            class="text-left"
-                                            name="voucherName{{ entry.uniqueName }}"
+                                            [cssClass]="'form-control mat-field-border text-left'"
+                                            [name]="'voucherName{{ entry.uniqueName }}'"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html]:
```suggestion
                                            [name]="'voucherName' + entry.uniqueName"
```

File: apps/web-giddh/src/app/shared/aside-menu-recurring-entry/aside.menu.recurringEntry.component.html [hunk 3]
```diff
@@ -18,71 +18,77 @@ <h3 class="aside-title">{{ localeData?.recurring_account }}</h3>
...
                             [options]="intervalOptions"
+                            (selectedOption)="onSelectInterval($event)"
                         >
                             <ng-template #optionTemplate let-option="option">
-                                <div class="account-list-item">{{ option?.label }}</div>
-                                <div class="account-list-item font-12">{{ option?.value }}</div>
+                                <a *ngIf="option" href="javascript:void(0)" class="linking-item">
+                                    <div class="option-label">{{ option?.label }}</div>
+                                    <div class="option-value">
+                                        {{ option?.label }}
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/aside-menu-recurring-entry/aside.menu.recurringEntry.component.html]:
```suggestion
                                        {{ option?.value }}
```

File: apps/web-giddh/src/app/theme/form-fields/select-field/select-field.component.html [hunk 4]
```diff
@@ -1,21 +1,57 @@
...
+                    [ngTemplateOutletContext]="{ option: option }"
+                    [ngTemplateOutlet]="optionTemplate"
+                ></ng-template>
+                <ng-container *ngIf="!optionTemplate">
+                    <span class="option-label"> {{ option?.label }}</span>
+                    <span class="option-value" *ngIf="showValueInLabel">{{ option.additional?.uniqueName }}</span>
+                </ng-container>
+            </mat-option>
+            <mat-option disabled *ngIf="fieldFilteredOptions?.length === 0">No results found</mat-option>
+            <mat-option *ngIf="fieldFilteredOptions?.length === 0 && showCreateNew">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/form-fields/select-field/select-field.component.html:50]: check this

**Review by the-prateek-mishra — APPROVED** [review]

File: apps/web-giddh/src/app/voucher/components/voucher-gst-treatment/voucher-gst-treatment.component.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        </span>
+    </div>
+
+    <div class="gst-treatment-body">
+        <form name="gstTreatmentForm" novalidate="">
+            <div class="container-fluid mr-t1">
+                <div>
+                    <div class="form-group">
+                        <label>{{ localeData?.party_type }}</label>
+                        <sh-select [options]="[]" [showClear]="false"></sh-select>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/voucher/components/voucher-gst-treatment/voucher-gst-treatment.component.html]: useless component

File: apps/web-giddh/src/app/voucher/components/voucher-print-inplace/voucher-print-in-place.component.ts [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+                    window.print();
+                }, 200);
+            }
+        }
+    }
+
+    /**
+     * Releases memory
+     *
+     * @memberof ProformaPrintInPlaceComponent
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/voucher/components/voucher-print-inplace/voucher-print-in-place.component.ts]: change name

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**